nixos/modules/programs/environment.nix

```diff
@@ -42,6 +43,7 @@ in
         GTK_PATH = [ "/lib/gtk-2.0" "/lib/gtk-3.0" ];
         XDG_CONFIG_DIRS = [ "/etc/xdg" ];
         XDG_DATA_DIRS = [ "/share" ];
+        XCURSOR_PATH = [ "/share/icons" ];
         MOZ_PLUGIN_PATH = [ "/lib/mozilla/plugins" ];
```

Comment:
Well yes, that was my thinking too. There is already a TODO somewhere here that much of it is out of place. Probably it would be better if all desktops startup scripts use the desktop-specific environment definitions from a common place (or display managers?) . But I can't myself work on refactoring this.